### PR TITLE
[4.0] Fix to modal not opening on edit button click

### PR DESCRIPTION
**Pull request by Arpit-24**
Pull Request for Issue #23910  .

### Summary of Changes

Added a replacement of &quot; by " in javascript to prevent invalid javascript code of document.getElementById(&quot;jform_request_id_id&quot;).value from running.

### Testing Instructions

Create a menu item of the Single article type
Click button "Edit" for editing selected article.
npm ci is required to compile the js file

### Expected result

Modal for edit article opens

### Actual result

Didn't happen earlier, happens after this commit.

### Documentation Changes Required

None

Screenshot (1256)
Screenshot (1257)

Results after this commit.

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 079cbe47b64e779ca4891f2b0b6c8493b633718c<br /><br />Good find!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

File: build/media_source/legacy/js/bootstrap-init.es5.js [hunk 1]
```diff
@@ -125,6 +125,8 @@ Joomla = window.Joomla || {};
 						if ($self.data('iframe').indexOf("document.getElementById") > 0){
 							var iframeTextArr = $self.data('iframe').split('+');
 							var idFieldArr = iframeTextArr[1].split('"');
+
```
Comment (infograf768) [inline build/media_source/legacy/js/bootstrap-init.es5.js]: Please take off tabs

Comment (Arpit-24) [reply build/media_source/legacy/js/bootstrap-init.es5.js]: Done.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on b869b9ac493d61b2135164a151b8c5929d33b9f7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on b869b9ac493d61b2135164a151b8c5929d33b9f7<br /><br />Same Behaviour with or without Patch (Patchtester used). <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

**Comment by infograf768** [comment]
@franz-wohlkoenig 
this pr needs npm ci

**Comment by ghost** [comment]
>  this pr needs npm ci

How to know this? Is this by Label "NPM Resource Changed"?

**Comment by ghost** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

**Comment by infograf768** [comment]
I guess so. But it is easy to know also because the file modified is a a js file in the build folder.
 `build/media_source/legacy/js/bootstrap-init.es5.js`

**Comment by ghost** [comment]
> But it is easy to know

this isn't true. Its easier for non-Devs to get Information in Test Instructions that a Pull Request can be tested by Patchtester or npm.

**Comment by infograf768** [comment]
Understand.
I guess we should systematically ask people proposing prs to state that npm is needed.

**Comment by ghost** [comment]
I'm willing to add a Sentence with Link how to use npm in every opening Thread who need it.

**Comment by Arpit-24** [comment]
Added npm ci required in the testing instructions.

**Comment by fastslack** [comment]
@jtester @test Patch applied and works as expected<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24349">issues.joomla.org/tracker/joomla-cms/24349</a>.</sub>

**Comment by brianteeman** [comment]
> How to know this? Is this by Label "NPM Resource Changed"?

Yes that's the easiest way

**Comment by roland-d** [comment]
FYI, since this is only a JavaScript change you can use `npm run build:js`. This is much faster than doing a complete cleanup with `npm ci`.

**Pull request merged**